server/server-kv-run.c

```diff
@@ -20,14 +20,24 @@
         http://beej.us/guide/bgnet/
 ****************************************/
 
+extern int flag_test;
+
+void
+process_server(char *message)
```

Comment:
The idea behind this implementation is that, separate server lib (server-kv.c & server-kv.h, which is only a lib without main func) and server internal test (server-kv-run.c which has main func) and the examples to run on server (in folder test named as server-kv-test), to make them as separate blocks, this flag here is to show if it is a internal test for server or run some other external examples/applications/test on the server.